wadas/domain/ai_model.py

```diff
@@ -60,7 +61,13 @@ def process_image(self, img_path, save_detection_image: bool):
 
         if not os.path.isfile(img_path):
             logger.error("%s is not a valid image path. Aborting.", img_path)
-            return
+            return None, None
+
+        try:
+            Image.open(img_path).verify()
+        except PIL.UnidentifiedImageError:
+            logger.error("%s is not a valid image file. Aborting.", img_path)
+            return None, None
 
         logger.info("Running detection on image %s ...", img_path)
         img = Image.open(img_path).convert("RGB")
```

Comment:
try:
            img = Image.open(img_path)
            img.verify()
        except PIL.FileNotFoundError:
            logger.error("%s is not a valid image path. Aborting.", img_path)
            return None, None
        except PIL.UnidentifiedImageError:
            logger.error("%s is not a valid image file. Aborting.", img_path)
            return None, None

        logger.info("Running detection on image %s ...", img_path)
        img = img.convert("RGB")